docs/pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-		 xmlns="http://maven.apache.org/POM/4.0.0"
-		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
```

Comment:
The namespace URI is treated only has an identifier. It is [not a goal](https://www.w3.org/TR/2009/REC-xml-names-20091208/#ns-decl) that it be directly usable for retrieval of a schema (if any exists). The URI should be exactly as documented. Changing the scheme to `https` actually makes it a different namespace name.

Reply:
Don't know that really matters here

Reply:
The guidance I've gotten is to replace them. The build works either way.